node/network/availability-distribution/src/lib.rs

```diff
@@ -123,8 +136,33 @@ impl AvailabilityDistributionSubsystem {
 				FromOverseer::Communication {
 					msg: AvailabilityDistributionMessage::AvailabilityFetchingRequest(req),
 				} => {
-					answer_request_log(&mut ctx, req, &self.metrics).await
+					answer_chunk_request_log(&mut ctx, req, &self.metrics).await
+				}
+				FromOverseer::Communication {
+					msg: AvailabilityDistributionMessage::PoVFetchingRequest(req),
+				} => {
+					answer_pov_request_log(&mut ctx, req, &self.metrics).await
 				}
+				FromOverseer::Communication {
+					msg: AvailabilityDistributionMessage::FetchPoV{
+                        relay_parent,
+                        from_validator,
+                        candidate_hash,
+                        tx,
+                    },
+				} => {
+                    log_error(
+                        pov_requester.fetch_pov(
```

Comment:
The previous API I think was better for backing - request the PoV from _someone_. Now backing has to worry about from whom. And if it fails, retry, for another validator.

Reply:
True, we could make pov distribution also listen to statements, that would be the cleanest approach to avoiding that, I think. Although, I have already implemented retry logic in backing now.